### PR TITLE
version: hardcode argv0 for runc compatibility

### PR DESCRIPTION
**Pull request by AkihiroSuda**
Signed-off-by: Akihiro Suda <suda.akihiro@lab.ntt.co.jp>

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
cc @giuseppe

**Comment by giuseppe** [comment]
just out of curiosity, what issue were you having with the previous version?

In any case, LGTM, feel free to merge it :-)

**Comment by AkihiroSuda** [comment]
> just out of curiosity, what issue were you having with the previous version?

It seemed hard to parse the version.
